runtime/src/lib.rs

```diff
@@ -1096,20 +1205,53 @@ impl membership::Config for Runtime {
     type WeightInfo = membership::weights::SubstrateWeight<Runtime>;
     type ReferralCutMaximumPercent = ReferralCutMaximumPercent;
     type CandidateStake = CandidateStake;
+    type DefaultMemberInvitesCount = DefaultMemberInvitesCount;
 }
 
 parameter_types! {
     pub const MaxCategoryDepth: u64 = 6;
-    pub const MaxSubcategories: u64 = 40;
-    pub const MaxThreadsInCategory: u64 = 20;
-    pub const MaxPostsInThread: u64 = 20;
-    pub const MaxModeratorsForCategory: u64 = 20;
+    pub const MaxSubcategories: u64 = 40; // TODO: adjust
```

Comment:
Why are there TODOs here?

Reply:
It's explained in the PR description. It's because https://github.com/Joystream/joystream/issues/4275 is not yet implemented, but the https://github.com/Joystream/joystream/issues/4300 proposal assumes it is

Reply:
4275 is now implemented by Ignazio in https://github.com/Joystream/joystream/pull/4352